spring-boot-security-taglibs/README.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
You can probably just remove this file as Grzegorz typically comes around and adds these after.